cmf-thud-freescale.xml

```diff
@@ -30,5 +30,5 @@
     <project name="meta-gplv2" path="meta-gplv2" revision="thud" remote="yocto-cgit"/>
     <project name="meta-rust/meta-rust" path="meta-rust" revision="master" remote="github"/>
 
-    <project name="andrey-konovalov/meta-freescale" path="meta-freescale" remote="github" revision="mmwg-master" />
+    <project name="linaro-mmwg/meta-freescale" path="meta-freescale" remote="github" revision="mmwg-master" />
```

Comment:
Maybe we will stick to initial revision of linaro-mmwg/meta-freescale (same as now) unless warrior patches doesn't cause any issue.

Reply:
it is mainly just backported gcc8 fixes for the imx op-tee port to enable it to compile OK, so I think it should be OK

Reply:
oh ok.

Reply:
LGTM. Build and image verified